client/blocks/plugins-scheduled-updates/schedule-form-paths.tsx

```diff
@@ -113,6 +121,11 @@ export function ScheduleFormPaths( props: Props ) {
 				) }
 			</div>
 			{ newPathError && <Text className="validation-msg">{ newPathError }</Text> }
+			{ MAX_SELECTABLE_PATHS === paths.length && (
+				<Text className="info-msg">
+					{ translate( 'You reached the maximum number of paths.' ) }
+				</Text>
+			) }
```

Comment:
Instead of showing that, could we just not show another input field? Kind of like we do with the Add Schedule button?

Reply:
That's how it works now... we don't show the last field.

In one of the previous PRs, I got feedback that the user might need to figure out why he can not add another, and additional info could be cool to be there.

Reply:
I see, thanks.

In the section description, could we replace "Do you want to add additional paths?" with "Add up to 5 additional paths to test:" instead? Feels more elegant.

Reply:
Sure, I'll replace the description.